packages/docz-core/src/Entries.tsx

```diff
@@ -114,6 +114,9 @@ export class Entries {
     const entries = await Promise.all(files.filter(isEntry).map(createEntry))
 
     for (const entry of entries) {
+      if (process.platform === "win32") {
+        entry.filepath = entry.filepath.split('\\').join('/')
```

Comment:
I think you can use `path.normalize` https://nodejs.org/api/path.html#path_path_normalize_path

```
❯ node -p "require('path').win32.normalize('/Users/test')"
\Users\test

Reply:
I tried with `path.normalize` and even `path.win32.normalize` but it doesn't work, strange 😞 

![image](https://user-images.githubusercontent.com/5435657/41381942-448fbc26-6f40-11e8-8cb4-c5b5d0d44beb.png)

Reply:
Seems that the tpl.js compiler just somehow mess up with the final import file with the double slash "\\" on windows

Reply:
https://github.com/pedronauck/docz/blob/master/packages/docz-core/templates/imports.tpl.js#L3

here?

Reply:
Yes, it's all fine even on the last phase of writing the import, but after writing the import file the slash disappear on windows and the Error appear.